infer/src/base/ProcessPool.ml

```diff
@@ -428,10 +490,11 @@ let create :
  fun ~jobs ~child_prologue ~f ~child_epilogue ~tasks ->
   let task_bar = TaskBar.create ~jobs in
   let children_pipes = create_pipes jobs in
+  let make_child = if true then spawn_child else fork_child in
```

Comment:
Heh. Make a config option? You can set a default based on Version.build_platform.

Reply:
Sure.

Reply:
What's the plan to choose between the two modes?

Reply:
Felt through the cracks. On it. Do you have a name you like?